components/member-profile/index.js

```diff
@@ -1,29 +1,29 @@
-import { getDataURL, getImgURL } from 'helper-functions/urls';
-
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from './member-profile.module.scss';
-import useFetch from 'custom-hooks/useFetch';
 
 const Profile = (props) => {
-  const imgLink = getImgURL(props.id);
-  const { data } = useFetch(getDataURL(props.id));
+  const memberName = `${props.membersData.first_name || '--'} ${
+    props.membersData.last_name || '--'
+  }`;
   return (
     <>
-      <img src={imgLink} className={classNames.profilePic} alt="profile" />
-      <div className={classNames.memberName}>
-        {data ? `${data['first_name']} ${data['last_name']} ` : `${props.id}`}
-      </div>
+      <img src={props.imageLink} className={classNames.profilePic} alt="profile" />
+      <div className={classNames.memberName}>{memberName}</div>
     </>
   );
 };
 
 Profile.propTypes = {
-  id: PropTypes.string
+  imageLink: PropTypes.string
+};
+
+Profile.propTypes = {
+  membersData: PropTypes.object
 };
 
 Profile.defaultProps = {
-  id: ''
+  imageLink: ''
```

Comment:
defaultProp for `membersData` is missing. Either make `membersData` as required or provide a defaultProp.